parser_test.go

```diff
@@ -100,7 +100,7 @@ func (s *testParserSuite) TestSimple(c *C) {
 		"max_rows", "min_rows", "national", "quarter", "escape", "grants", "status", "fields", "triggers", "language",
 		"delay_key_write", "isolation", "partitions", "repeatable", "committed", "uncommitted", "only", "serializable", "level",
 		"curtime", "variables", "dayname", "version", "btree", "hash", "row_format", "dynamic", "fixed", "compressed",
-		"compact", "redundant", "sql_no_cache sql_no_cache", "sql_cache sql_cache", "action", "round",
+		"compact", "redundant", "`sql_no_cache`", "`sql_cache`", "action", "round",
```

Comment:
```suggestion
		"compact", "redundant", "1 sql_no_cache`", "1 sql_cache", "action", "round",
```

quoting defeats the whole purpose of this test.